### ISSUE TITLE
Form-Message Color Contrast

### ISSUE THREAD

**Issue by Zauberbutter** [bigfishtv/turret#30]
The colors of `.form-message` elements, have an insufficient color contrast:

https://webaim.org/resources/contrastchecker/?fcolor=CC0000&bcolor=FFFFFF
```css
.form-message.error {
    color: #c00
}
```

https://webaim.org/resources/contrastchecker/?fcolor=FF5500&bcolor=FFFFFF
```css
.form-message.warning {
    color: #f50
}
```

https://webaim.org/resources/contrastchecker/?fcolor=00B300&bcolor=FFFFFF
```css
.form-message.success {
    color: #00b300
}
```

https://webaim.org/resources/contrastchecker/?fcolor=0088CC&bcolor=FFFFFF
```css
.form-message.info {
    color: #08c
}
```

Maybe the *-800 colors could be used there?

Edit: Same problem with buttons.

**Comment by Zauberbutter** [comment]
@scottdejonge Are you open for PR's on this? How should we handle this, maybe use *-800 colors for stuff like this or create a whole new color-scheme?
